### DISCUSSION TITLE
download from https://

### DISCUSSION THREAD

**Discussion by johnoo** [doublecmd/doublecmd#1384] [General]
Hi,
can I download files from https? 
and password protected?
If there is plugin for it please suggest. Or Howto
my issue is that:
wget -user=username -password=pw1234 https://domain.com/folder1/file1 works 
but not sure how to download all files from dir or suggestions I found on web did not help. Being able to do it with doublecmd would be great.

**Comment by alexx2000** [comment]
Hi, under Windows you can try [HTTP Browser](http://totalcmd.net/plugring/http2.html) plugin.

**Reply by johnoo** [reply]
Thank you a and what about linux?

**Comment by alexx2000** [comment]
I haven't heard that something like this exists on Linux.
